### DISCUSSION TITLE
Resize dropdown menu

### DISCUSSION THREAD

**Discussion by hawwokitty** [React95/React95#463] [Q&A] [answered]
Hi, is it possible to make the dropdown menu longer?

**Comment by ggdaltoso** [comment, marked as answer]
That's the same problem with the Modal we're talking about earlier at #459 but this fix is a lot easier.

This is what we need to do to extend styles on Dropdown component via `className`:

```diff
export const Dropdown = forwardRef<HTMLSelectElement, DropdownProps>(
  ({ options = defaultOptions, ...rest }, ref) => {
    const { style, otherProps, className } = sprinkles(rest);

    return (
-      <Frame style={style} className={cn(wrapper, className)}>
+       <Frame style={style} className={cn(wrapper, className, otherProps.className)}>
        <select {...otherProps} className={cn(select)} ref={ref}>
          {options &&
            options.map(option => (
              <option key={option} value={option}>
                {option}
              </option>
            ))}
        </select>
      </Frame>
    );
  },
);
```

I also have one more thing I'd like to improve on Dropdown. 
Anyway, would you like to contribute?

**Reply by hawwokitty** [reply]
ohhh I see, thanks! 
I'm actually new to github.. what does it mean to "contribute" in this context? 😳

**Reply by ggdaltoso** [reply]
You can fix this for this library. If you're okay with it, I can guide you.

**Reply by hawwokitty** [reply]
omg yeah i would love to try! i would need guidance for sure tho, i havent edited the library itself when working with it, cuz i want to be able to update it and not have everything break XD

**Reply by ggdaltoso** [reply]
[Try to follow our contributing guide first](https://github.com/React95/React95/blob/master/CONTRIBUTING.md). If any questions pop up, you can ping me and I'll be more than glad to help you.

**Reply by hawwokitty** [reply]
okay so i tried to follow the guide, however i dont really know 100% what im doing, i forked it and now i have a copy in my repositories, is that right? Also when i pushed the change with github desktop, suddenly it says i now have 19 new changes in the packages/.../components.snap? but when i click them they say "No content changes found" 😟 whats up with that? should i not have used github desktop? 🤔

**Reply by ggdaltoso** [reply]
> i forked it and now i have a copy in my repositories, is that right?

Yes! That's fine! This is how it works when you don't have permission to write to a repo: you fork into your account and then you can do whatever you want in your version. As soon as you finish working on the changes in your repo, you can open a pull request to the origin repo.

> Also when i pushed the change with github desktop, suddenly it says i now have 19 new changes in the packages/.../components.snap? but when i click them they say "No content changes found" 😟 whats up with that?

 1. Are you using Windows? If so, it could be related to filemode|autocrlf config. However, I also found [this issue with the GitHub desktop app](https://github.com/desktop/desktop/issues/3987).
 2. I don't have experience with the GitHub desktop app. I usually use the terminal or [VSCode source control panel](https://code.visualstudio.com/docs/sourcecontrol/overview). Could you try using the terminal to see if you get the same output?

**Reply by hawwokitty** [reply]
Okay I tried to just do the commit with git through VScode, but I still got those changes to the snapshots after i push it to github 🤔 Maybe it's normal? I've never really done this so idk.
Also should I add my [portfolio](https://hawwokitty.github.io/my-portfolio/) to the "Awesome projects" list? XD I don't want to intrude or anything, but that would be very cool haha.

Another issue I'm having right now is that when i try to run the localhost, none of the components actually show up, they just keep loading forever, so i don't really know how to check if what I added even works...

**Reply by ggdaltoso** [reply]
I just checked [your commit](https://github.com/React95/React95/commit/ab6c1da5fc0954a11d959f0c84bf950cd8bf54c1) and want to say thanks for following our guides. [You are ready to create the PR](https://github.com/React95/React95/compare/master...hawwokitty:React95:master) 🎉 

> Also should I add my [portfolio](https://hawwokitty.github.io/my-portfolio/) to the "Awesome projects" list?

YOU MUST. It is a wonderful portfolio and a must-have for the React95 project's list!

> Another issue I'm having right now is that when i try to run the localhost, none of the components actually show up, they just keep loading forever, so i don't really know how to check if what I added even works...

It takes a while to boot everything. When the browser opens you see the storybook preview loading and all stories on the left, right? It can take some time to load everything. React95 is a big library because of its components, icons, and clippy packages. In addition to that, if you see any weird error popping up in the devtools, please share it with me. We should not have any problems running storybook.

**Reply by hawwokitty** [reply]
yay! will create the PR tomorrow most likely :D gotta add the portfolio first! 

And yeah, i see the storybook preview loading and everything, its just the components themselves that arent loading, i'll try to just let them load for a really long time to see if that fixes it o7
